tests/Aspire.Elastic.Clients.Elasticsearch.Tests/AspireElasticClientExtensionsTest.cs

```diff
@@ -0,0 +1,146 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Elastic.Clients.Elasticsearch;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Aspire.Elastic.Clients.Elasticsearch.Tests;
+
+public class AspireElasticClientExtensionsTest : IClassFixture<ElasticsearchContainerFixture>
```

Comment:
Can we also add a test similar to this? https://github.com/dotnet/aspire/blob/3406514607bb775cb5c9693c269832ca5c5a4170/tests/Aspire.NATS.Net.Tests/AspireNatsClientExtensionsTests.cs#L178-L204

@eerhardt would that make sense here?

Reply:
Yes, I think that would be a good to have this test.